WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -434,7 +442,15 @@ private extension NoticePresenter {
         }
 
         @objc func updateFrame(notification: Notification) {
-            frame = MaskView.calculateFrame(parent: parentView, untouchableVC: untouchableVC)
+            // Update the `frame` on the next run loop. When this Notification event handler is
+            // called, the `self.parentView` still has the frame from the previous orientation.
+            // Running this routine after the current run loop ensures we have the correct frame.
+            DispatchQueue.main.async { [weak self] in
```

Comment:
This is just a suggestion but, what if we wrapped the `DispatchQueue.main.async` call in a method with a name that explained what was going on, such as
```swift
waitALoopSoRotationWorksOnSimulator() {
    self.frame = MaskView.calculateFrame(parent: self.parentView, untouchableVC: self.untouchableVC)
}
```

Reply:
I'm not sure about this. 😬I think it's already well contained in this method. Perhaps I can add a comment in line 73 to re-evaluate this method if the iPad simulator issue ever gets fixed.

Reply:
What made me think of it is that we're calling `DispatchQueue.main.async` twice, with the only hint that they're related being the comments. I like good comments, but I fear that they're fragile, ya know?

Anyway, still up to you 👍